### PR TITLE
add mazestudio jocker

### PR DESCRIPTION
**Pull request by mazestd**
## Description

add new board "mazestudio jocker"

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by mazestd — COMMENTED** [review]
Updating Bootmagic Config

File: keyboards/mazestudio/jocker/info.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        "pid": "0x4A01",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
```
Comment (waffle87) [inline keyboards/mazestudio/jocker/info.json:19]:
```suggestion
    },
    "dynamic_keymap": {
        "layer_count": 3
    },
```

File: keyboards/mazestudio/jocker/config.h
Comment (waffle87) [inline keyboards/mazestudio/jocker/config.h:1]: This file should be removed with the above suggestion.

File: keyboards/mazestudio/jocker/info.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    "bootmagic":{
+        "enabled": true,
```
Comment (waffle87) [inline keyboards/mazestudio/jocker/info.json:21]:
```suggestion
```

File: keyboards/mazestudio/jocker/info.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    "maintainer": "mazestd",
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "vid": "0x70F5",
+        "pid": "0x4A01",
+        "device_version": "0.0.1"
+    },
+    "features": {
```
Comment (waffle87) [inline keyboards/mazestudio/jocker/info.json:13]:
```suggestion
    "features": {
        "bootmagic": true,
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by mazestd — COMMENTED** [review]
Just updated based on your review @waffle87 . Thanks

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mazestudio/jocker/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Jocker 1800
+
+![jocker1800]( https://i.imgur.com/aQEiE4H.jpg )
```
[the remark this answers is not in the document]
Comment (mazestd) [reply keyboards/mazestudio/jocker/readme.md]: Thanks @fauxpark , but may i know why i should add h at the end of the url?

[the remark this answers is not in the document]
Comment (fauxpark) [reply keyboards/mazestudio/jocker/readme.md]: https://api.imgur.com/models/image#thumbs

**Review by drashna — APPROVED** [review]

**Pull request merged**